Signal/src/Models/TSMessageAdapaters/TSAnimatedAdapter.m

```diff
@@ -131,8 +132,14 @@ - (BOOL)canPerformEditingAction:(SEL)action
 - (void)performEditingAction:(SEL)action
 {
     if (action == @selector(copy:)) {
+        NSString *utiType = [MIMETypeUtil utiTypeForMIMEType:self.attachment.contentType];
+        if (!utiType) {
+            OWSAssert(0);
+            utiType = (NSString *)kUTTypeVideo;
```

Comment:
Good catch, that's a mistake. But I'm not sure what makes sense as a default here since GIF will always be handled correctly by `[MIMETypeUtil utiTypeForMIMEType:]` and so it'll be wrong if that fails.